source/patterns/@aws-solutions-constructs/aws-lambda-dynamodb/test/integ.deployFunctionWithExistingVpc.ts

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Nice touch adding this test - looks like only the Sagemaker construct included this test up to this point.

Reply:
Hi @biffgaut I've created the sample application using the construct with a VPC and validated that the Lambda function was able to write to the DDB table.